Bugzilla/Constants.pm

```diff
@@ -702,7 +695,7 @@ sub _bz_locations {
         # The script should really generate these graphs directly...
         'webdotdir'      => "$datadir/webdot",
         'extensionsdir'  => "$libpath/extensions",
-        'assetsdir'      => "$datadir/assets",
+        'assetsdir'      => "$libpath/assets",
```

Comment:
libpath is effectively the web root, and datadir is the NFS-mounted partition. I think this comment is about how confusing it is to use 'libpath' for this -- I think may I mean 'cgipath'... The whole nomenclature of bz_locations() is hairy though. 

Switching this means assets are per-web head, which is a huge improvement.